README.md

```
@@ -32,13 +32,31 @@ Forky is a command-line interface (CLI) tool that implements a git-style convers
 
 ## Usage
 
+### CLI
 Run the CLI application:
-
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)